runtime/plugin/nvim.lua

```diff
@@ -7,17 +7,26 @@ vim.api.nvim_create_user_command('Inspect', function(cmd)
 end, { desc = 'Inspect highlights and extmarks at the cursor', bang = true })
 
 vim.api.nvim_create_user_command('InspectTree', function(cmd)
+  ---@type string
+  local lang = cmd.fargs and cmd.fargs[1]
   if cmd.mods ~= '' or cmd.count ~= 0 then
     local count = cmd.count ~= 0 and cmd.count or ''
     local new = cmd.mods ~= '' and 'new' or 'vnew'
 
     vim.treesitter.inspect_tree({
       command = ('%s %s%s'):format(cmd.mods, count, new),
+      lang = lang,
     })
   else
-    vim.treesitter.inspect_tree()
+    vim.treesitter.inspect_tree({
+      lang = lang,
+    })
```

Comment:
This could be simplified to:
```lua
  if cmd.mods ~= '' or cmd.count ~= 0 then
    local count = cmd.count ~= 0 and cmd.count or ''
    local new = cmd.mods ~= '' and 'new' or 'vnew'

    vim.treesitter.inspect_tree({
      command = ('%s %s%s'):format(cmd.mods, count, new),
      lang = cmd.fargs and cmd.fargs[1],
    })
  else
    vim.treesitter.inspect_tree()
```

Reply:
Hmm... in my testing I couldn't get this to work; something like `:InspectTree markdown_inline` will still just show the regular markdown tree

Reply:
Markdown is special; you'll _never_ have `markdown_inline` root tree.